Fix compatibility with WooCommerce 10.8.0 beta (#5390)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

xxxx-xx-xx - version 10.7.0

> * Fix - Fix compatibility with WooCommerce 10.8 checkout-evidence guard (required created-via in orders)

Reviewer: Non-blocking nit: We could simplify the description.
```suggestion
* Fix - Fix compatibility with WooCommerce 10.8 payment handling, where created-via is required for orders
```
